guacamole-common-js/src/main/webapp/modules/Keyboard.js

```diff
@@ -39,6 +39,13 @@ Guacamole.Keyboard = function(element) {
     var guac_keyboard = this;
 
     /**
+     * Flag indicating whether the browser is running on Mac
+     *
+     * @type Boolean
+     */
+    var is_mac = (navigator && navigator.platform && navigator.platform.match(/^mac/i));
```

Comment:
Mind also removing this `is_mac` flag? I'm worried someone might eventually ... use it.

We were grudgingly forced to detect the OS in commit b6769e4, as there was no other way to accurately handle `Alt` on Macs (see [GUAC-391](https://glyptodon.org/jira/browse/GUAC-391)), but the last thing we want to encourage is browser sniffing.